### PR TITLE
Updates task requests api

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
Date: 21 November 2023
Developer Name: @Ajeyakrishna-k 

----

## Description: 
Update the task requests api schema as the existing api's.

### CODE REVIEW

File: task-requests/README.md [hunk 1]
```diff
@@ -6,34 +6,37 @@
...
 |               Route                |    Description    |
 | :--------------------------------: | :---------------: |
-|      [GET /task-requests](#get-task-requests)      | Return all task requests |
-|      [GET /task-requests/:id](#get-task-requests-id)      | Return task request details for the provided id |
-|     [POST /task-requests](#post-task-requests)     | Creates new task request  |
-|     [PATCH /task-requests/:id](#patch-task-requests-id) |   Updates task request   |
+|      [GET /taskRequests](#get-task-requests)      | Return all task requests |
+|      [GET /taskRequests/:id](#get-task-requests-id)      | Return task request details for the provided id |
+|     [POST /taskRequests](#post-task-requests)     | Creates new task request  |
+|     [PATCH /taskRequests](#patch-task-requests) |   Updates task request   |
```
Comment (heyrandhir) [inline task-requests/README.md:37]: It is recommended to use either kebab case or snake case when naming your APIs, but camel case should be avoided.

File: task-requests/README.md [hunk 2]
```diff
@@ -181,17 +182,20 @@ Content: {
...
+  - Required: `action=[approve | reject]` (`action` action either approves or rejects the task request of the given id)
 - **Headers**  
   Content-Type: application/json
 - **Body** 
 ```
-	content: { <task_request_object> }
+	content: {
+			taskRequestId : <id>
+			userId : <userId>
+			}
```
Comment (heyrandhir) [inline task-requests/README.md:198]: can you check the formatting . parentheses seems out of place.

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: task-requests/README.md (hunk 1 above)
Comment (Ajeyakrishna-k) [reply task-requests/README.md:37]: These api names were previously approved and present in backend repo before I started working on these. It should be fixed with this issue: 
- https://github.com/Real-Dev-Squad/website-backend/issues/1597

File: task-requests/README.md (hunk 2 above)
Comment (Ajeyakrishna-k) [reply task-requests/README.md]: Done.

**Review by heyrandhir — APPROVED** [review]

**Pull request merged**